Fix the `check_unix' function, which looks for `/bin/uname' to determine
whether we're on a Unix-like system.

--- doxygen-1.5.8/tmake/bin/tmake	2008-12-06 14:16:20.000000000 +0100
+++ doxygen-1.5.8/tmake/bin/tmake	2009-03-05 11:29:55.000000000 +0100
@@ -234,17 +234,7 @@ sub tmake_verb {
 #
 
 sub check_unix {
-    my($r);
-    $r = 0;
-    if ( -f "/bin/uname" ) {
-	$r = 1;
-	(-f "\\bin\\uname") && ($r = 0);
-    }
-    if ( -f "/usr/bin/uname" ) {
-	$r = 1;
-	(-f "\\usr\\bin\\uname") && ($r = 0);
-    }
-    return $r;
+    return 1;
 }
 
